{{cookiecutter.repo_slug}}/.github/labels.yml

```diff
@@ -25,41 +25,41 @@
   description: "Generic maintenance tasks."
 - name: "ci"
   color: 1d76db
-  description: "Work that improves the continue integration."
+  description: "Work that improves the continuous integration."
 - name: "dependencies"
   color: 1d76db
-  description: "Upgrade or downgrade of project dependencies."
+  description: "Change in project dependencies."
```

Comment:
It seems like this would also be an appropriate label for adding or removing dependencies -- in which case it's not about upgrade or downgrade, just change.